src/playbooks/_flavor_features/metadata.obsah.yaml

```diff
@@ -0,0 +1,13 @@
+---
+variables:
+  flavor:
+    help: Base flavor to use in this deployment.
+  features:
+    parameter: --add-feature
```

Comment:
We could also consider adding a `--list-feature` option to display visible features, but that can be done outside this PR, wdyt?